Resources/doc/empty-content-status-code.rst

```diff
@@ -0,0 +1,27 @@
+Status code when responding with no content
+===========================================
+
+In some use cases the api should not send any content, especially when deleting (*DELETE*) or updating (*PUT* or *PATCH*) a resource.
+
+In order to provide a status code that should be sent when having no content, the configuration must be customized:
```

Comment:
That's not true, ``204`` is the default status code, the configuration should only be used to use another status code.